lighthouse-cli/bin.ts

```diff
@@ -65,6 +65,7 @@ const cliFlags = yargs
     'port'
   ], 'Configuration:')
   .describe({
+    'disable-cache-clearing': 'Disable clearing of cache and storage before the run',
```

Comment:
`disable-storage-reset` comes to mind. I actually think being more generic than "cache" is good for the reasons you mention. It applies to all forms of storage on the origin.